### ISSUE TITLE
Archive the repo?

### ISSUE THREAD

**Issue by cpcloud** [ibis-project/ibis-examples#30]
Should we archive this repo? It seems like we're not actively developing this and doing these sorts of things in other places.

**Comment by jcrist** [comment]
Yeah, makes sense to me. ~Will do.~ *(edit: actually, I lack permissions to do this)*
